graphql/schema/types/host.graphql

```diff
@@ -44,6 +44,11 @@ type Host {
   distro: DistroInfo
   distroId: String
   elapsed: Time # running task start time
+  events(
+    limit: Int = 0
+    page: Int = 0
+    sortDir: SortDirection = DESC
```

Comment:
If you want to laugh. This one line here took me 2 days to figure out because I initially had it as ASC and it would keep defaulting to the wrong order. 🤦‍♂️